packages/frontend-vitals-monitoring/README.md

```diff
@@ -0,0 +1,33 @@
+# @govuk-one-login/frontend-vital-signs
+
+## Purpose
+
+@govuk-one-login/frontend-vital-signs is a set of functions to monitor performance on front end repositories
+
+## Table of Contents
+
+1. [Installation](#installation)
+2. [How to use](#how-to-use)
+3. [Issues](#issues)
+
+## Installation
+
+Add to your project using `npm i @govuk-one-login/frontend-vital-signs`
+
+## How to use
+
+### createPersonalDataHeaders
```

Comment:
I think there's a bit here that's still from the old passthrough-headers package.